packages/engines/src/index.ts

```diff
@@ -59,28 +58,6 @@ export { enginesVersion } from '@prisma/engines-version'
  */
 
 path.join(__dirname, '../query-engine-darwin')
-path.join(__dirname, '../introspection-engine-darwin')
-
-path.join(__dirname, '../query-engine-darwin-arm64')
-path.join(__dirname, '../introspection-engine-darwin-arm64')
-
-path.join(__dirname, '../query-engine-debian-openssl-1.0.x')
-path.join(__dirname, '../introspection-engine-debian-openssl-1.0.x')
-
-path.join(__dirname, '../query-engine-debian-openssl-1.1.x')
-path.join(__dirname, '../introspection-engine-debian-openssl-1.1.x')
-
-path.join(__dirname, '../query-engine-debian-openssl-3.0.x')
-path.join(__dirname, '../introspection-engine-debian-openssl-3.0.x')
-
-path.join(__dirname, '../query-engine-rhel-openssl-1.0.x')
-path.join(__dirname, '../introspection-engine-rhel-openssl-1.0.x')
-
-path.join(__dirname, '../query-engine-rhel-openssl-1.1.x')
-path.join(__dirname, '../introspection-engine-rhel-openssl-1.1.x')
-
-path.join(__dirname, '../query-engine-rhel-openssl-3.0.x')
-path.join(__dirname, '../introspection-engine-rhel-openssl-3.0.x')
```

Comment:
Not sure why the `query-engine` lines were removed here?
Maybe not needed 🤔 , though we could keep it safe and keep them and figure this out another time?

Reply:
Thanks for spotting this, I've recovered the mentions to the query-engine